### DISCUSSION TITLE
What ist the OEP definition for a framework and a model?

### DISCUSSION THREAD

**Discussion by MichaelGeFr** [OpenEnergyPlatform/examples#215] [Q&A]
Could somebody please provide me with a **OEF conform definition of a framework and a mode**l? I do not understand where the difference is made on the OEP. For example the model **Balmorel** is also present as a framework. 
I know the different **factsheets** but I am not confident in deciding whether I am using a model or a framework.

**Comment by jh-RLI** [comment]
Sorry for the late response. Discussions are quite undercover on GitHub if they are not reviewed closely.

The definition for the difference between model & framework is in the objective. While the model is described in a formal way the framework implements the model with specific scenario definitions using a certain software some community or organization might develop. 

We also have these definitions in the OpenEnergyOntology:
http://openenergyplatform.org/ontology/oeo/OEO_00000172/
http://openenergyplatform.org/ontology/oeo/OEO_00000277/
